pkg/logs/input/journald/tailer.go

```diff
@@ -28,12 +28,13 @@ const (
 
 // Tailer collects logs from a journal.
 type Tailer struct {
-	source     *config.LogSource
-	outputChan chan *message.Message
-	journal    *sdjournal.Journal
-	blacklist  map[string]bool
-	stop       chan struct{}
-	done       chan struct{}
+	source          *config.LogSource
+	outputChan      chan *message.Message
+	journal         *sdjournal.Journal
+	blacklistSystem map[string]bool
+	blacklistUser   map[string]bool
```

Comment:
"Denylist" is the generally-preferred term for this these days.  In the spirit of leaving the codebase better than you found it, this PR would be a good place to make that change (or maybe just `userExclude` and `systemExclude`?)

Reply:
Thanks. I think I'll create a new PR but will keep this in mind